packages/terra-embedded-content-consumer/src/terra-dev-site/doc/example/DataStatusConsumer.jsx

```diff
@@ -11,6 +11,10 @@ const createListItem = (itemName) => {
 
 const appendLifeCycleStatuses = (statuses) => {
   const frame = document.getElementById('data-embedded-consumer-data-status');
+  if (!frame) {
+    return;
+  }
+
```

Comment:
this likely needs added to the test example as well

Reply:
https://github.com/cerner/terra-framework/pull/615/commits/f968f0995f741c93b931f83364d8303db3ca8b93